### DISCUSSION TITLE
How to redirect / to baseUrl

### DISCUSSION THREAD

**Discussion by jeanmatthieud** [nitrojs/nitro#1764] [Q&A]
Hi,

I'm using Nuxt3, with a baseUrl.
I see that Nitro still respond to the root / request.
How can I redirect this root request to the baseUrl ?

Thanks for your help

**Comment by pi0** [comment]
Hi. Do you mind to move this to a discussion/issue in nuxt repository with a reproduction and ping me? 🙏🏼
